index.bs

```diff
@@ -701,17 +784,28 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 				1. If |r| is not "granted", then reject |p| with a "NotAllowedError" DOMException
 
-				1. Let |cleanItemList| be an empty sequence&lt;{{Blob}}>.
+				1. Let |itemList| and |cleanItemList| be an empty sequence&lt;{{Blob}}>.
+
+				1. For each |itemData| in |data|:
+
+					1. For each |item| in |itemData|:
+						1. If [=Promise=] to {{Blob}} in |item| is rejected, then throw "Promises to Blobs were rejected." DOMException.
```

Comment:
What if `item` contained a `DOMString`? Somewhere a conversion to a `Blob` would have to happen.

Reply:
Oops, missed this one. Yes, the conversion from DOMString to Blob needs to happen inside in this step